webapp/application.py

```diff
@@ -126,6 +134,7 @@ def application_page(token):
     return flask.render_template(
         "applications/application.html",
         title="You application",
+        withdrawal_reasons=withdrawal_reasons,
```

Comment:
@nottrobin I keep getting this error from here. Do you know why by any chance? 

![image](https://user-images.githubusercontent.com/57550290/185340094-46aaf8d0-f195-4f02-9834-175eebd2e9fe.png)

Reply:
sorry, fixed! :)

Reply:
oh sorry. I imagine you needed `.items()` or something?

Reply:
yep I added `.item()`